log/log.go

```diff
@@ -41,31 +40,51 @@ var levelPrefix = [...]string{
 var (
 	// Level stores the current logging level.
 	Level = LevelInfo
-	// SysLogger is a syslog Writer to be used if not nil.
-	SysLogger *syslog.Writer
+	// outputInterface stores a SetLogger() parameter.
+	outputInterface Interface
 )
 
+// Interface specifies the necessary methods for an alternate output
+// destination passed in via SetLogger.
+//
+// Interface is satisfied by syslog.Writer.
+type Interface interface {
```

Comment:
`log.Interface` is not a good name for a `syslog.Writer` interface. Maybe `SyslogWriter`?

Nitpick: it's satisfied by the pointer type `*syslog.Writer`